### DISCUSSION TITLE
feature(github): Issue/PR templates

### DISCUSSION THREAD

**Discussion by DjDeveloperr** [harmonyland/harmony#42] [Ideas]
**Suggestion**
There could be Issue and Pull Request templates.

**Why is this needed?**
Makes issues and pull requests follow a standard which can be more informational than just description - IMO makes job of maintainer/s easier.

**Comment by 6days9weeks** [comment]
+1 from me. Cause this will make the Pull Request and issues cuter (I want a cute template) but any works.

**Comment by tejasag** [comment]
+1 from me too.. Will make the devs work easier and they will be able to quickly find the major issues from the less important ones...

**Comment by AkiaCode** [comment]
I think

ex)  
```
issues 
ㄴ bug
ㄴ  question
ㄴ enhancement
ㄴ other
PR
ㄴ bugfix
ㄴ enhancement
ㄴ  fix typo
ㄴ other
```

**Comment by AkiaCode** [comment]
Link: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates

**Comment by Helloyunho** [comment]
Issue templates are done. Time to make pr template!
